src/main/scala/sbtassembly/Shader.scala

```diff
@@ -81,7 +81,16 @@ private[sbtassembly] object Shader {
     }}
 
     val proc = JJProcessor(jjrules, verbose = level == Level.Debug, true)
-    val files = AssemblyUtils.getMappings(dir, Set())
+
+    /*
+    jarjar MisplacedClassProcessor class transforms byte[] to a class using org.objectweb.asm.ClassReader.getClassName
+    which always translates class names containing '.' into '/', regardless of OS platform.
+    We need to transform any windows file paths in order for jarjar to match them properly and not omit them.
+     */
+//    val files = AssemblyUtils.getMappings(dir, Set())
+
+    val files = AssemblyUtils.getMappings(dir, Set()).map(f =>
+      if (f._2.contains('\\')) (f._1, f._2.replace('\\', '/')) else f)
```

Comment:
Yea. I was thinking the latter, like

```scala
val PathList(xs) = f._2
xs.mkString("/")
```

Not sure if it work but at least you'd then have backslash handling in one spot.